.github/workflows/provisioning-tests.yaml

```diff
@@ -0,0 +1,58 @@
+name: Provisioning tests
+
+on:
+  push: 
+    branches:
+      - 'dev-v*'
+      - 'release-v*'
+  pull_request:
+    branches:
+      - 'dev-v*'
+      - 'release-v*'
+
+jobs:
+  provisioning-test:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    container: 
+      image: rancher/dapper:v0.6.0
+      options: --privileged
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        dist: [rke2, k3s]
+        k8s-minor: [25, 26, 27, 28]
```

Comment:
```suggestion
        k8s-minor: [25, 26, 27, 28, 29]
```

Reply:
k8s 1.29 is not yet supported in rancher.. so avoided adding it just yet..

Reply:
@brandond WDYT about adding this now as opposed to later? I feel like although CI isn't quite stable yet, this could be useful to identify potential issues with 1.29 we haven't encountered yet and we can always force merge (we have been anyway).

Reply:
added both 29 and 30 and removed 25 and 26 as per supported version list